site/src/component/GradeDist/GradeDist.scss

```diff
@@ -75,4 +75,26 @@
   .pie-text {
     font-size: 1em;
   }
+}
+@media only screen and (max-width: 750px) {
+  .pie-text {
+    font-size: 1.5em;
+  }
+}
```

Comment:
Not needed. Media query below is for max-width: 749px so this one only affects screen widths of exactly 750px. Font size is also already 1.5em by default for larger screen widths.
```suggestion
```

Reply:
I see. I removed 750 and modified the 749px to 1319px because it seems like the the pie cut off the text starting from there. Thanks for the suggestions. I deployed the change, you can check it out and see if there is anything else I need to change.

Reply:
Looks good. I'm going to go ahead and resolve this conversation.